tests/src/test/resources/projects/PantsExample/fastpass/bin/fastpass

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+COURSIER="$(dirname $0)/coursier"
+exec $COURSIER launch fastpass --contrib -- "$@"
```

Comment:
with this setup we won't be able to control the version of fastpass, but I think we can be ok with this for now